modules/userman/http.go

```diff
@@ -0,0 +1,165 @@
+package userman
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"strings"
+	"time"
+
+
+	"github.com/gorilla/mux"
+)
+
+// HandleProfile returns the handler for fetching single user profile
+func (m *Module) HandleProfile() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		// Create a context of execution
+		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+		defer cancel()
+
+		// Get the path parameters
+		vars := mux.Vars(r)
+		project := vars["project"]
+		dbType := vars["dbType"]
+		id := vars["id"]
+
+		// Get the JWT token from header
+		tokens, ok := r.Header["Authorization"]
+		if !ok {
+			tokens = []string{""}
+		}
+		token := strings.TrimPrefix(tokens[0], "Bearer ")
+
+		status, err, result := m.Profile(ctx, token, dbType, project, id)
```

Comment:
Handle the error immediately after this line. Only have the condition, `err != nil`. We don't need an else if you return in the if itself